src/parser/jobs/gnb/modules/GnashingFang.tsx

```diff
@@ -59,15 +53,6 @@ export class AmmoCombo extends Analyser {
 
 	override initialise() {
 		super.initialise()
-
-		const RELEVANT_ACTIONS = [this.data.actions.JUGULAR_RIP.id, this.data.actions.ABDOMEN_TEAR.id, this.data.actions.EYE_GOUGE.id, this.data.actions.HYPERVELOCITY.id]
```

Comment:
Why is all of this getting nuked?

Reply:
Moved to Continuation which uses CoreProcs.

"Continuation which is the core proc module for all "Ready to " actions + Lion Heart"